docs/site/todo-tutorial-controller.md

```diff
@@ -26,148 +26,44 @@ For more information about Controllers, see
+? What is the name of the model to use with this CRUD repository? Todo
+? What is the name of your CRUD repository? TodoRepository
+? What is the type of your ID? number
+? What is the base HTTP path name of the CRUD operations? /todos
+   create src/controllers/todo.controller.ts
+   update src/controllers/index.ts
+
+Controller todo was created in src/controllers/
 ```
 
-The `@repository` decorator will retrieve and inject an instance of the
-`TodoRepository` whenever an inbound request is being handled. The lifecycle of
-controller objects is per-request, which means that a new controller instance is
-created for each request. As a result, we want to inject our `TodoRepository`
-since the creation of these instances is more complex and expensive than making
-new controller instances.
+Let's review the `TodoController` located in
+`src/controllers/todo.controller.ts`. The `@repository` decorator will retrieve
+and inject an instance of the `TodoRepository` whenever an inbound request is
+being handled. The lifecycle of controller objects is per-request, which means
+that a new controller instance is created for each request. As a result, we want
+to inject our `TodoRepository` since the creation of these instances is more
+complex and expensive than making new controller instances.
 
 > **NOTE**: You can customize the lifecycle of _all_ bindings in LoopBack 4!
 > Controllers can easily be made to use singleton lifecycles to minimize startup
 > costs. For more information, see the
 > [Dependency injection](Dependency-injection.md) section of our docs.
 
-Now that the repository is connected, let's create our first controller
-function.
-
-#### src/controllers/todo.controller.ts
-
-```ts
-import {repository} from '@loopback/repository';
-import {TodoRepository} from '../repositories';
-import {Todo} from '../models';
-import {HttpErrors, post, param, requestBody} from '@loopback/rest';
-
-export class TodoController {
-  constructor(@repository(TodoRepository) protected todoRepo: TodoRepository) {}
-
-  @post('/todos')
-  async createTodo(@requestBody() todo: Todo) {
-    if (!todo.title) {
-      throw new HttpErrors.BadRequest('title is required');
-    }
-    return await this.todoRepo.create(todo);
-  }
-}
-```
-
-In this example, we're using two new decorators to provide LoopBack with
-metadata about the route, verb and the format of the incoming request body:
+In this example, there are two new decorators to provide LoopBack with metadata
+about the route, verb and the format of the incoming request body:
 
 - `@post('/todos')` creates metadata for `@loopback/rest` so that it can
   redirect requests to this function when the path and verb match.
 - `@requestBody()` associates the OpenAPI schema for a Todo with the body of the
   request so that LoopBack can validate the format of an incoming request
   (**Note**: As of this writing, schematic validation is not yet functional).
```

Comment:
Can we remove the `Note` stating `schematic validation` isn't available yet since it is now?